sdk/src/schema.ts

```diff
@@ -101,3 +101,8 @@ export type ContractOptions = {
   refTime: number
   proofSize: number
 }
+
+export interface Result<T, E> {
```

Comment:
There was discussion about this topic before. All in all, we decided to follow the TypeScript convention for handling errors. Therefore:
1. Always return an "ok" result without any wrapper.
2. Do not add `Error` to the function signature.
3. Instead of returning an error, throw it, and then use the catch statement

Based on these decisions, this interface should not be used